ofl/merienda/METADATA.pb

```diff
@@ -1,26 +1,27 @@
 name: "Merienda"
-designer: "Eduardo Tunni"
+designer: "Eben Sorkin"
```

Comment:
@emmamarichal copy/paste error between yaml files?

Reply:
Oh yes, I thought I had corrected it, I may have overwritten the correction in the last update